.ruby-gemset

```diff
@@ -0,0 +1 @@
+hasbeen-in
```

Comment:
Killer feature of RVM. If you also want to use it with rbenv you have to install a plugin AFAIK.